File: app/src/main/java/com/bignerdranch/android/movielist/ActivitySplashScreen.java
package com.bignerdranch.android.movielist;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;

public class ActivitySplashScreen extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        getSupportActionBar().hide();
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                //This method will be executed once the timer is over
                // Start your app main activity
                Intent i = new Intent(ActivitySplashScreen.this, ActivityMain.class);
                startActivity(i);
                // close this activity
                finish();
            }
        }, 3000);
    }
}

File: app/src/main/java/com/bignerdranch/android/movielist/ActivitySplashScreen.kt
package com.bignerdranch.android.movielist

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.os.Handler

class ActivitySplashScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        supportActionBar!!.hide()
        Handler().postDelayed({ //This method will be executed once the timer is over
            // Start your app main activity
            val i = Intent(this@ActivitySplashScreen, ActivityMain::class.java)
            startActivity(i)
            // close this activity
            finish()
        }, 3000)
    }
}